AnalyticsService/app/routes.py

```diff
@@ -4,14 +4,18 @@
 
 import os
 from flask import Blueprint
-from app.models import Budget, db
+from app.models import Budget, StripeAccount, db
 from flask import Blueprint, jsonify, request, abort, make_response
 from app.decorators import token_required
 from app.utils import decode_jwt
-
+import stripe
+from stripe.error import StripeError
 
 routes = Blueprint('routes', __name__)
 
+#Set Stripe API Key
+stripe.api_key = "sk_test_51O4uCWFy63ZKr0XemD3A1rCloE3Su65QRVkFPIWiQ5wgemqAUJOSOJtoSeKVxkKpSXRdMjWp0nRn31rvtWrQ26sY00ffEdglnR"
```

Comment:
lets not commit the api key in the future. I would recommend changing your api key since it has been pushed to github, or you can overwrite this commit and push.